### PR TITLE
require name when copying app

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?237822

@biyeun

### CODE REVIEW

File: corehq/apps/app_manager/views/apps.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def copy_app_check_domain(request, domain, name, app_id_or_source):
 def copy_app(request, domain):
     app_id = request.POST.get('app')
     form = CopyApplicationForm(
-        app_id, request.POST, export_zipped_apps_enabled=toggles.EXPORT_ZIPPED_APPS.enabled(request.user.username)
+        domain, app_id, request.POST, export_zipped_apps_enabled=toggles.EXPORT_ZIPPED_APPS.enabled(request.user.username)
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/apps.py]: E501 line too long (122 > 115 characters)

**Comment by biyeun** [comment]
@orangejenny 
potentially related build failure? failing on domain transfer?

```
FAIL: test_permissions_for_activation (corehq.apps.domain.tests.test_domain_transfer.TestTransferDomainViews)
```

Also looks like it might potentially be resolved by a rebase w/ master?

**Comment by orangejenny** [comment]
Don't think it's related; that test fails pretty frequently. Merged master, let's see if that fixes it.

**Comment by biyeun** [comment]
ok will merge then

**Comment by nickpell** [comment]
This PR broke the build - the failure in the most recent commit is real.

**Pull request merged**